core/state/statedb.go

```diff
@@ -88,14 +89,18 @@ func (self *StateDB) Refund(gas *big.Int) {
 	self.refund.Add(self.refund, gas)
 }
 
-/*
- * GETTERS
- */
-
 func (self *StateDB) HasAccount(addr common.Address) bool {
 	return self.GetStateObject(addr) != nil
 }
 
+func (self *StateDB) Exist(addr common.Address) bool {
+	return self.GetStateObject(addr) != nil
```

Comment:
Yes, I agree. I'll follow up with another PR. I do not wish to introduce any more changes.